README.md

```diff
@@ -12,7 +12,7 @@ Beluga is an extensible C++17 library with a ground-up implementation of the Mon
 - Emphasis on the prevention of regressions and facilitation of code improvements through test coverage.
 - Semi-automated benchmarks that can be used to validate different configurations.
 
```

Comment:
Sorry for the intrusion:

Maybe makes sense to add a note here with a link to the andino project if you want(Already public). Just to evidence that this is working with a public robot and anyone can see the specs of the robot (which SBC is quite humble (Raspi))

https://github.com/Ekumen-OS/andino

Reply:
In the context of the video of course

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/239/commits/580c34ee540cd5a98ae6e79fa8215af7fb337ace!